watcher/src/bin/migrate.rs

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! A utility for exporting the history of Attestation Verification
+//! Reports generated by MobileCoin consensus nodes
+
+use clap::Parser;
+use hex::ToHex;
+use mc_attest_core::VerificationReport;
+use mc_blockchain_types::{AVRHistoricalRecord, AVRHistoricalRecords};
+use mc_common::logger::{create_app_logger, o};
+use mc_crypto_keys::Ed25519Public;
+use mc_watcher::{error::WatcherDBError, watcher_db::WatcherDB};
+use std::{fs, path::PathBuf};
+use url::Url;
+
+/// Command line configuration.
+#[derive(Parser)]
+#[clap(
+    name = "mc-watcher-avr-export",
+    about = "A utility for exporting the history of mobile consensus enclave AVRs"
+)]
+pub struct Config {
+    /// Path to watcher db (lmdb).
+    #[clap(
+        long,
+        default_value = "/tmp/watcher-db",
+        parse(from_os_str),
+        env = "MC_WATCHER_DB"
+    )]
+    pub watcher_db: PathBuf,
+
+    /// Path for the avr-history.toml bootstrap file to land to land
+    #[clap(
+        long,
+        default_value = "/tmp/watcher/avr-history.toml",
```

Comment:
How about we default to the current dir?

```suggestion
        default_value = "avr-history.toml",
```